CalSmokeApp/Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
 gem 'calabash-cucumber', :github => "calabash/calabash-ios", :branch => "develop"
-gem "cucumber", "2.99.0"
+gem "cucumber", "~>2.0"
```

Comment:
Why do we need to do this? in my understanding `cucumber` version should not be changed